Gruntfile.js

```diff
@@ -4,14 +4,17 @@ module.exports = function( grunt ) {
 
 	var sass = require( 'node-sass' );
 
+	var processors = [ require( 'autoprefixer' )() ];
+	if ( grunt.option('env') === 'production' ) {
```

Comment:
Would this make the most sense? compared to the regular `NODE_ENV` we use in most projects?
This is a loosely held opinion.

Reply:
I don't think `NODE_ENV` is available inside grunt. At least, I couldn't get it working.